pkg/signup/signup_callback_handler_test.go

```diff
@@ -0,0 +1,45 @@
+package signup_test
+
+import (
+	"github.com/codeready-toolchain/registration-service/pkg/configuration"
+	"github.com/codeready-toolchain/registration-service/pkg/signup"
+	"github.com/gin-gonic/gin"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+)
+
+func TestSignupCallbackHandler(t *testing.T) {
+	// Create a request to pass to our handler. We don't have any query parameters for now, so we'll
```

Comment:
can we be consistent with our comments? seems like some of them start with upper-case and others start with lower case..